CHANGELOG.rst

```diff
@@ -12,6 +12,9 @@ Change Log
 Unreleased
 ----------
 
+* Add swagger support
```

Comment:
This is in somehow a breaking change, maybe this should be specified here or once we will create the next feature

Reply:
I thought the actual breaking change was made in #126.

Reply:
Yes, but AFAIK in that PR there are not changes that are incompatible with the previous version of api-doc-tools